File: 2058-ConcatenationOfArray/2058-ConcatenationOfArray.java
// Last updated: 9/14/2025, 9:09:11 AM
class Solution {
    public int[] getConcatenation(int[] nums) {
        int len = nums.length;
        int []ans = new int[len*2];

        int i = 0;
        int j = len; 
        while(j<ans.length){
            ans[i] = nums[i];
            ans[j] = nums[i];

            i++;
            j++;
        }

        return ans;
    }
}

File: 2058-ConcatenationOfArray/2058-ConcatenationOfArray.cpp
// Last updated: 9/14/2025, 9:09:05 AM
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        // int size = nums.size();
        vector<int> ans;
        for(int i=0;i<nums.size();i++){
            ans.push_back(nums[i]);
        }
        for(int i=0;i<nums.size();i++){
            ans.push_back(nums[i]);
        }

        return ans;
    }
};